package.json

```diff
@@ -1,15 +1,19 @@
 {
-  "name": "@momento/sdk",
+  "name": "@gomomento/sdk",
   "version": "0.0.1",
   "description": "Client SDK for Momento services",
   "main": "dist/src/index.js",
   "files": [
     "dist/src"
   ],
   "types": "dist/src/index.d.ts",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/momentohq/client-sdk-javascript"
+  },
   "scripts": {
     "test": "jest",
-    "integration": "jest --config jest-integration.config.js",
+    "integration": "jest --config jest-integration.config.js --detectOpenHandles --maxWorkers 1",
```

Comment:
just want to check to make sure you want to keep this setting (some changes I know are from debugging the build errors)

Reply:
Good point. Will remove those.